GOVERNANCE.md

```diff
@@ -136,26 +173,40 @@ issue, but must recuse themselves from voting on the issue.
 * Must be core contributor for at least one year
 
 #### Nominations
-* Nominations are taken over a public github issue ticket over the course of 2 weeks.
+* Nominations are taken over a public GitHub issue ticket over the course of 2 weeks.
 * Only Core Contributors may nominate folks
 * Self Nominations are allowed
 * At the conclusion of the 2 weeks, the list of nominations is posted on the ticket and this ticket is closed.
 
 
 #### Election Process
 
-* Voting occurs over a period of at least 1 week, at the conclusion of the nominations. Voting is blind and mediated by either an application or a third party like Numfocus.
-Each voter can vote zero or more times, once per each candidate. As this is not about ranking but about capabilities, voters vote on a yes/no basis per candidate -- “would I trust this person to lead ArviZ?”.
-* In the event of a tie there will be a runoff election for the tied candidates. To avoid further ties and discriminate more among the tied candidates, this vote will be held by [Majority Judgment](https://en.wikipedia.org/wiki/Majority_judgment): for each candidate, voters judge their suitability for office as either Excellent, Very Good, Good, Acceptable, Poor, or Reject. Multiple candidates may be given the same grade by a voter. The candidate with the highest median grade is the winner.
-* If more than one candidate has the same highest median-grade, the MJ winner is discovered by removing (one-by-one) any grades equal in value to the shared median grade from each tied candidate's total. This is repeated until only one of the previously tied candidates is currently found to have the highest median-grade.
-* If ties are still present after this second round, the winner will be chosen at random. Each person tied will pick a number between 0 and 100, and a random integer will be generated from random.org. The person with the closet circular distance will be selected.
+* Voting occurs over a period of at least 1 week, at the conclusion of the nominations.
+  Voting is blind and mediated by either an application or a third party like NumFOCUS.
+  Each voter can vote zero or more times, once per each candidate. As this is not about
+  ranking but about capabilities, voters vote on a yes/neutral/no basis
```

Comment:
I modified to a yes/neutral/no basis, not the yes/no type of voting we did for the first election. Now the following question arises.

> Do we interpret yes=+1, neutral=0 and no=-1 and leave the point below unchanged?
> Or do we use the same 2 threshold pattern?

In the second option, candidate face "only themselves" in the sense that the results of other candidates do not affect them being chosen -> there is no possibility for ties. We may have to restrict the min and max number of council members though, otherwise we could face a situation where nobody gets chosen.

Moreover, in any of the 2 cases we need some rules on how to modify the size of the council.

Reply:
I propose having the following steps:

* yes/neutral/no voting
   - if the number of accepted candidates is in the interval [4, 13] (or whatever) election results are definitive (unless challenged)
   - otherwise we do yes=1, neutral=0, no=-1 and rank
* If we get here it means we are in the +1,0,-1 alternative so we can keep points from 2 onwards as they are now to break ties.

Reply:
> - yes/neutral/no voting
        - if the number of accepted candidates is in the interval [4, 13] (or whatever) election results are definitive (unless challenged)

This looks good to me, with Majority Judgment to break potential ties. This is simple, focused on capabilities instead of ranking, and is basically an extension of what we did for the first election, extended with a "neutral" option

Reply:
@AlexAndorra I am not sure I understand the comment, mainly due to the quote of the 1a step. Assuming we chose 4, 13 as thresholds for max and min number of council members:
* if there are 15 candidates and 12 get yes>=60% and no<=20%, all 12 are directly elected as council members. 
* if instead all 15 get yes>=60% and no<=20%, they would then be ranked by interpreting yes=1, neutral=0, no=-1 and the first 13 would be elected council members
   - if there were ties (e.g. 12, 13 and 14 candidates have all a score of 4) the 2nd election would be triggered exactly as it is written now:

> In the event of a tie there will be a runoff election for the tied candidates. To avoid
  further ties and discriminate more among the tied candidates, this vote will be held
  by [Majority Judgment](https://en.wikipedia.org/wiki/Majority_judgment) (MJ): for each
  candidate, voters judge their suitability for office as either Excellent, Very Good,
  Good, Acceptable, Poor, or Reject. Multiple candidates may be given the same grade by
  a voter. The candidate with the highest median grade is the winner.